File: CS50x/2. Arrays/8.c
/*

> Date Created: July 20, 2023
> Author: Ishaan Rastogi
> Purpose: To print ASCII value of characters of 7.c's bunch of characacters
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working

/*

Strings are of variable length.
A string is just an array of characters.
String contains n+1 bytes where n is the length of string. That extra last byte is \0. This \0 is called NUL.

*/

#include<stdio.h>

int main()
{
    char c1 = 'H';
    char c2 = 'I';
    char c3 = '!';

    printf("%i %i %i\n", c1, c2, c3); // %i for int

}

/* 

Ctrl + Shift + `

Terminal Code Loading...

gcc 8.c
./a.exe

NOTE- Whenever you are using cs50 functions like get_string, make sure to follow the YouTube video named 'how to use cs50 library in vscode' till
the end, uploaded by a channel named Mahmoud Saad in the year 2022. Don't miss any single second.

*/

File: Web Development Projects/15 Intermediate JS/8.js
/*
> Date Created: June 10, 2024
> Author: Ishaan Rastogi
> Purpose: To create a program called FizzBuzz
> Operating System: This is only for Windows OS, it may or may not work on other OS
> Program Status: 100% Working
*/

/*

WAP that prints the numbers from 1-100. But for multiples of 3, print "Fizz" instead of the number & for the multiples of 5, print "Buzz". For numbers which are multiples of both 3 & 5, print "FizzBuzz".

*/

var output = [];
var c = 1; //count

//Creating function to push new numbers
function fizzBuzz() {
    output.push(c);
    c++; // c = c + 1
    console.log(output);
}

if ( c % 3 === 0) {
    output.push("Fizz");
}
else if ( c % 5 === 0) {
    output.push("Buzz");
}
else if ( c % 3 === 0 && c % 5 === 0) {
    output.push("FizzBuzz");
}
else {
    output.push(count);
}

/* 
For Terminal Code... Ctrl + Shift + `
*/